docs/getting_started.rst

```diff
@@ -1,5 +1,3 @@
-.. _getting_started:
```

Comment:
I think it's useful to put labels at least for major sections, so that they can easily be linked to from other parts of the docs or via Intersphinx:
http://sphinx-doc.org/markup/inline.html#cross-referencing-arbitrary-locations